httpserver/swarm.cpp

```diff
@@ -195,15 +200,6 @@ swarm_id_t get_swarm_by_pk(const std::vector<SwarmInfo>& all_swarms,
     return cur_best;
 }
 
-std::vector<sn_record_t> Swarm::other_nodes() const {
```

Comment:
We can return by const reference now. Should be changed accordingly on the other side (in `push_message`).

Reply:
Good catch.
Updated in last commit.